app/src/main/java/io/gnosis/safe/ui/transactions/execution/TxAdvancedParamsFragment.kt

```diff
@@ -0,0 +1,32 @@
+package io.gnosis.safe.ui.transactions.execution
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import io.gnosis.safe.ScreenId
+import io.gnosis.safe.databinding.FragmentTxEditAdvancedParamsBinding
+import io.gnosis.safe.di.components.ViewComponent
+import io.gnosis.safe.ui.base.fragment.BaseViewBindingFragment
+
+class TxAdvancedParamsFragment : BaseViewBindingFragment<FragmentTxEditAdvancedParamsBinding>() {
+
+    override fun screenId() = ScreenId.TRANSACTIONS_EXEC_REVIEW_ADVANCED
+
+    override fun inject(component: ViewComponent) {
+        component.inject(this)
+    }
+
+    override fun inflateBinding(
+        inflater: LayoutInflater,
+        container: ViewGroup?
+    ): FragmentTxEditAdvancedParamsBinding =
+        FragmentTxEditAdvancedParamsBinding.inflate(inflater, container, false)
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        with(binding) {
+
```

Comment:
Can this with(binding) be removed?

Reply:
it's a placeholder for future pr. The whole fragment is empty right now.